src/liballoc/slice.rs

```diff
@@ -490,7 +490,7 @@ impl [u8] {
     ///
     /// To lowercase the value in-place, use [`make_ascii_lowercase`].
     ///
-    /// [`make_ascii_lowercase`]: #method.make_ascii_lowercase
+    /// [`make_ascii_lowercase`]: ../../std/primitive.slice.html#method.make_ascii_lowercase
```

Comment:
Why would you need to change this?

Reply:
I'm trying to fix Travis errors caused by linkchecker. 

Current rustdoc renders all the methods that come from `Deref::Target` type. That means all the available methods from `[Cell<T>]` will be rendered in `Cell`'s document. Whereas, these methods' document contain several relative path links. These relative paths can work in `primitive.slice.html` but not work in `Cell.html`.